test/images/cloudbuild.yaml

```diff
@@ -1,8 +1,8 @@
 # See https://cloud.google.com/cloud-build/docs/build-config
 
 # this must be specified in seconds. If omitted, defaults to 600s (10 mins)
-# the timeout was increased because some images might take longer to build (e.g.: sample-apiserver, pets/redis-installer)
-timeout: 5400s
+# the timeout was increased because some images might take longer to build (e.g.: sample-apiserver, pets/redis-installer, volume/rbd)
+timeout: 10800s
```

Comment:
What impact could this have on costs / cloud bill?

Reply:
well, not every image takes this much of time but we for some images like `volume/rbd` hence this is just a timeout and doesn't mean that every job will take this much amount of time.